packages/sitecore-jss/src/feaas/themes.ts

```diff
@@ -21,17 +21,19 @@ type RevisionType = 'staged' | 'published';
  */
 const FEAAS_LIBRARY_ID_REGEX = /-library--([^\s]+)/;
 
-export const FEAAS_SERVER_URL = 'https://feaas.blob.core.windows.net';
+export const FEAAS_SERVER_URL_STAGING = 'https://feaasstaging.blob.core.windows.net';
```

Comment:
Optional improvement:
Do you want to define an _Enum_?
```ts
enum FEAAS_SERVER_URL = {
  STAGING: '...',
  BETA: '...',
  PROD: '...'
}
```

Reply:
Going to leave it since this is temporary anyway (will be removed once we switch to file proxy).

Reply:
Makes sense to me, right